### DISCUSSION TITLE
Is It Advisable to add multiple wallets to a profile

### DISCUSSION THREAD

**Discussion by reallordmicky** [WalletConnect/walletconnect-monorepo#5443] [Developer Support]
My questions basically is, is advisable to add different wallets address to a registration profile with one recipient address and not be flag as a Sybil. Thanks for an open discussion

**Comment by Inikesta** [comment]
It is really your choice though. The project is backed by binance and so it is relatively safe

**Comment by Vineetkg20** [comment]
yep i don't think there is any problem

**Comment by adenolabashiru** [comment]
You can add multiple wallets since it's from the team
